### ISSUE TITLE
Per method throttling config is not being deleted on redeployment

### ISSUE THREAD

**Issue by lkolchin** [DianaIonita/serverless-api-gateway-throttling#5]
Hi,

I've deployed the config per method and verified that it was deployed.
But when I removed it from serverless.yaml and left Throttling config as a Global setting - The per method config was not deleted on redeployment.

Before:
```
custom:
  serverless-jetpack:
    mode: npm # (default `yarn`)
  # Configures throttling settings for all http endpoints
  apiGatewayThrottling:
    maxRequestsPerSecond: 1000
    maxConcurrentRequests: 500

  landingPage1:
    role: landingPageRole
    handler: src/handler1.landingPage
    package:
      exclude:
        - "**/**"
      include:
        - src/handler1.js
    events:
      - http:
          path: 'landingPage1'
          method: get
          cors: true
          throttling:
            maxRequestsPerSecond: 2000
            maxConcurrentRequests: 1000
    environment:
      BG_COLOUR: ${self:custom.backgroundColour.${self:provider.stage}} 
```

After:
```
custom:
  serverless-jetpack:
    mode: npm # (default `yarn`)
  # Configures throttling settings for all http endpoints
  apiGatewayThrottling:
    maxRequestsPerSecond: 1000
    maxConcurrentRequests: 500

  landingPage1:
    role: landingPageRole
    handler: src/handler1.landingPage
    package:
      exclude:
        - "**/**"
      include:
        - src/handler1.js
    events:
      - http:
          path: 'landingPage1'
          method: get
          cors: true
    environment:
      BG_COLOUR: ${self:custom.backgroundColour.${self:provider.stage}} 
```

**Comment by DianaIonita** [comment]
Hi @lkolchin,

Thanks for flagging this, it's indeed an issue. I'm having some trouble getting throttling settings to be inherited from the stage again, hopefully someone [might be able to help](https://stackoverflow.com/questions/61431295/configure-an-api-gateway-method-to-inherit-throttling-settings-from-the-stage).

**Comment by scadu** [comment]
Hi,

I wonder if we could set `-1` as default for the method throttling settings instead of trying to remove it?
I see that's how it's done in [AWS Terraform provider](https://github.com/hashicorp/terraform-provider-aws/blob/3b002dfdfaa672c7e2527c1dfd322723b394e624/aws/resource_aws_api_gateway_method_settings.go#L67-L76):
```go
"throttling_burst_limit": {
	Type:     schema.TypeInt,
	Optional: true,
	Default:  -1,
},
"throttling_rate_limit": {
	Type:     schema.TypeFloat,
	Optional: true,
	Default:  -1,
},
```

Seems that the AWS provider [has been fixed not that long time ago](https://github.com/hashicorp/terraform-provider-aws/commit/969a889f38e94c7ff3d8fb4348d2303b952ee9bd#diff-31095c2857f20f08140178b063518617d42eaeea6aed6dcbd639f36d22ddebd9).

**Comment by DianaIonita** [comment]
Should now be fixed in [v1.1.0](https://github.com/DianaIonita/serverless-api-gateway-throttling/releases/tag/1.1.0).
